### PR TITLE
Add kotlin extension / Update README with sample code

### PR DESCRIPTION
**Pull request by wisemuji**
Major changes:
* Add kotlin extension
* Update README
* Add sample code

Minor changes
* Update dependencies
* Bump version to 1.0.1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,11 +15,21 @@ Install library with Gradle
 ### Gradle:
 Add this to your project level build.gradle:
 
-TBD
+```groovy
```
Comment (kimdoori) [inline README.md:26]: I see....

File: README.md [hunk 2]
```diff
@@ -1,9 +1,17 @@
-[ProjectGithubUrl]:     https://github.com/spoqa/LongPressListener
-[PlatformBadge]:  https://img.shields.io/badge/Platform-Android-blue.svg
+[ProjectGithubUrl]: https://github.com/spoqa/LongPressListener
+[PlatformBadge]: https://img.shields.io/badge/Platform-Android-blue.svg
+[LicenceUrl]: https://opensource.org/licenses/MIT
+[LicenseBadge]: https://img.shields.io/badge/License-MIT-blue.svg
+[BintrayUrl]: https://android-arsenal.com/details/1/7134
+[BintrayBadge]: https://img.shields.io/bintray/dt/spoqa/maven/LongPressListener.svg
```
Comment (kimdoori) [inline README.md:6]: wow..

**Review by kimdoori — APPROVED** [review]
Nice!! 💯

**Comment by wisemuji** [comment]
Thank you ❤️

**Pull request merged**